package_policies/M1.md

```diff
@@ -1,14 +1,10 @@
-**M1.** Each xSDK-compatible package must support either the **community GNU Autoconf or CMake
-options**<sup>3</sup> as defined in the following document:
+**M1.** Each xSDK-compatible package must support portable installation through Spack.
 
-[xSDK Community Installation Policies: GNU Autoconf and CMake Options](./community-installation-policies.md).
+All configuration, build, and installation phases must be specified in a [Spack](https://spack.io/))
+package recipe.
 
-The goal of this policy is to enable individual users, computing centers, and package managers (such
-as [Spack](https://spack.io/)) to install the package in a way that is compatible with
-other xSDK packages on the same system. It is understood that managing all the dependencies for a variety
-of packages is nontrivial and is not resolved by standard options.
+Packages should have a build system that is appropriate for the language (e.g.~ CMake, Autoconf, setup.py), and the packages should attempt to follow the best practices and guidelines of the respective environment.
+For example, a proper configuration phase should be used where relevant (a list of platform specific makefiles would not be acceptable).
 
------
-
-<sup>3</sup> Note that the package is free to use any mechanism, for example Python as PETSc does, to implement
-the options but to users they should appear as if a CMake or GNU Autoconf configure package.
+The goal of this policy is to enable individual users and computing centers to install the package in a
```

Comment:
I would remove reference to `individual users and computing centers` . Perhaps it can be phrased differently to convey broad  and portable installs [i.e not restricted to a few special cases - which is possible with spack]

Reply:
I've modified the wording slightly